products/memgraph/upcoming/tutorials/analyzing-TED-talks.md

```diff
@@ -69,10 +69,10 @@ only during this run of Memgraph.
 
 ### Example Queries using OpenCypher
 
-In the queries below, we are using [OpenCypher](https://www.opencypher.org) 
+In the queries below, we are using [OpenCypher](https://www.opencypher.org)
 to query Memgraph via the console.
 
- 
+
```

Comment:
Please undo all unnecessary changes. The file does have formatting issues, but it isn't up to you to fix them in this diff.

Reply:
I only opened file with text editor, editor made formatting changes. Ok :+1: